test/runnable/cdcmp.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+src_file=${EXTRA_FILES}/${TEST_NAME}.d
+expect_file=${EXTRA_FILES}/${TEST_NAME}.out
+tmp_file=${RESULTS_TEST_DIR}/${TEST_NAME}.out
+obj_file=${RESULTS_TEST_DIR}/${TEST_NAME}.o
+
+if [ $OS == "linux" ] && [ $MODEL == "64" ]; then
+  $DMD -c -O ${src_file} -of${obj_file}
+  objdump --disassemble --disassembler-options=intel-mnemonic ${obj_file} | tail -n+3 > ${tmp_file}
+  diff ${expect_file} ${tmp_file}
```

Comment:
If you don't add `-p` here, you will complain in a few months time that the output mismatch on the auto-tester is horrible.

Reply:
I didn't see anything in the documentation about `-p` which would suggest that.

Reply:
`-p` is for the context of the diff. For additional lines around the diff, there's `-u`.
That's why we use `-pu` them for the other diff scripts:

https://github.com/dlang/dmd/blob/028229d1d6d75a8c85ea9afc1509056261bfe3f5/test/compilable/extra-files/json-postscript.sh#L6

There's another reason. It plays well with `AUTO_UPDATE` - see e.g. https://github.com/dlang/dmd/pull/8215

Reply:
I use `meld` which is a visual diff. It's not a problem.